typescript/utils/src/objects.ts

```diff
@@ -175,3 +179,35 @@ export function normalizeConfig(obj: WithAddress<any>): any {
 
   return obj;
 }
+
+// write a function that will go through an object and sort any arrays it finds
+export function sortValidatorsAndModulesInConfig(config: any): any {
```

Comment:
This file (and this whole package mostly) is for utilities agnostic to a particular concept or use case IMHO. So if you'd like to include this here, consider generalizing it to not be so specific to `validatorsAndModules`. Otherwise move it to the SDK

Reply:
@paulbalaji - I'm going to move `normalizeConfig` to the SDK as well given that it references an address